website/docs/cloud-docs/run/states.mdx

```diff
@@ -190,11 +190,7 @@ After applying, the run proceeds automatically to completion.
 
 ## The Post-Apply Stage
 
-The post-apply phase only occurs if the workspace has [run tasks](/terraform/cloud-docs/workspaces/settings/run-tasks) configured to begin after Terraform complete the apply operation. 
-
-Unlike other run task stages, the post-apply stage gathers information about the resulting response of an apply operation because that operation has already occurred, and HCP Terraform has already provisioned infrastructure. HCP Terraform sends the information about the run to the configured external system with the `passed` or `failed` status along with the configuration version of the run.
-
-Only confirmed runs can enter this phase.
+The post-apply phase only occurs if the workspace has [run tasks](/terraform/cloud-docs/workspaces/settings/run-tasks) configured to begin after Terraform successfully completes an apply operation. During this phase, HCP Terraform sends information about the run to the configured external system and waits for a `passed` or `failed` response. However, unlike other stages in the run task process, a failed outcome will not halt the run since HCP Terraform has already provisioned the infrastructure.
```

Comment:
Teeny suggestions for active voice and present tense:
```suggestion
The post-apply phase only occurs if you configure [run tasks](/terraform/cloud-docs/workspaces/settings/run-tasks) on a workspace to begin after Terraform completes an apply operation. During this phase, HCP Terraform sends information about the run to the configured external system and waits for a `passed` or `failed` response. However, unlike other stages in the run task process, a failed outcome does not halt the run since HCP Terraform has already provisioned the infrastructure.
```

Reply:
Done